exercises/reverse-string/description.md

```diff
@@ -0,0 +1,7 @@
+# Reverse String
+
+The objective is to write a function that reverses the input sequence of characters.
```

Comment:
@Insti would `routine` be better word instead of `function`?  Is function too specific?

Reply:
This was discussed a bit in the context of not writing "write a program to.."

https://github.com/exercism/problem-specifications/issues/321

The conclusion seemed to be: 
> go with a super generic "some code" to avoid the but this is not an X argument.

Reply:
I'm not sure what I should update for description, do you think it's not generic enough even with function? I was thinking of saying "create a way to reverse input sequence of characters", but the wording is kinda strange and too vague.

Reply:
What are the reasons, if any, that we should not write "Reverse a string" here?

If we get an answer to that question maybe it'll help us figure out what we should write.